Remotion/Core/UnitTests/Core.UnitTests.csproj

```diff
@@ -23,36 +23,32 @@
     <ProjectReference Include="..\Tools\Core.Tools.csproj" />
     <ProjectReference Include="..\..\Development\Core\Development.Core.csproj" />
     <ProjectReference Include="..\Core\Core.Core.csproj" />
+    <ProjectReference Include="..\..\Development\UnitTesting.IsolatedCodeRunner\Development.UnitTesting.IsolatedCodeRunner.csproj" />
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.Net.Compilers.Toolset" GeneratePathProperty="true" ExcludeAssets="all" />
+    <PackageReference Include="System.CodeDom" GeneratePathProperty="true" />
+    <PackageReference Include="System.Configuration.ConfigurationManager" GeneratePathProperty="true" />
   </ItemGroup>
   <ItemGroup>
     <None Update="Reflection\TestDomain\ContextAwareTypeUtilityTest\app.config">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-  <ItemGroup Condition="'$(IsDotNetFramework)' == 'false'">
-    <ProjectReference Include="..\..\Development\UnitTesting.IsolatedCodeRunner\Development.UnitTesting.IsolatedCodeRunner.csproj" />
-  </ItemGroup>
   <!-- BEGIN: ensure that required DLLs available in output directory for IsolatedCodeRunner to work correctly.
        Due to DotNet magic, some projects require that System.Configuration.ConfigurationManager and System.CodeDOM are copied locally while other projects 
        resovle it from the system directory. Until we can identify the underlying reason and switch IsolatedCodeRunner to use the system directory dependency,
        the deps.json file for IsolatedCodeRunner requires a local copy.
 
        NOTE: If the versions do not intersect, the file is always copied locally. This is the reason why no workaround is needed for .NET 6.
   -->
-  <ItemGroup Condition="'$(IsDotNetFramework)' == 'false'">
-    <PackageReference Include="System.CodeDom" GeneratePathProperty="true" />
-    <PackageReference Include="System.Configuration.ConfigurationManager" GeneratePathProperty="true" />
-  </ItemGroup>
   <Target Name="CopySystemConfigurationConfigurationManager" AfterTargets="ResolveAssemblyReferences" Condition="'$(PkgSystem_Configuration_ConfigurationManager)' != ''">
-    <ItemGroup Condition="'$(IsDotNetFramework)' == 'false'">
+    <ItemGroup>
       <ReferenceCopyLocalPaths Include="$(PkgSystem_Configuration_ConfigurationManager)/lib/net8.0/System.Configuration.ConfigurationManager.dll"/>
     </ItemGroup>
```

Comment:
This section needs to be restored to it's original ordering to preserve the comments set for the respective parts of the dependency list